index.bs

```diff
@@ -975,17 +1024,19 @@ return a point in time.
 
 <h3 algorithm id="obtaining-a-report">Obtaining a report</h3>
 
-To <dfn>obtain a report</dfn> given an [=attribution source=] |source| and an [=attribution trigger=] |trigger|:
+To <dfn>obtain a report</dfn> given an [=attribution source=] |source|, an [=attribution trigger=]
+|trigger|, and an [=event-level trigger configuration=] |config|:
```

Comment:
(see above regarding config)

Reply:
Because this algorithm needs to know which config is matching.